modules/orca/variables.tf

```diff
@@ -89,6 +89,12 @@ variable "orca_default_bucket" {
 }
 
 
+variable "orca_reports_bucket" {
```

Comment:
modules/orca/main.tf also needs to be updated

Reply:
It will eventually, but presently nothing consumes this value. We'll see what comes out of the automated report generation.

Reply:
Removed pending further discussion.